Quadriga/src/main/java/edu/asu/spring/quadriga/dto/ProjectStatisticsDTO.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package edu.asu.spring.quadriga.dto;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * This class represents the column mappings for project statistics settings
+ * table.
+ * 
+ * @author Ajay Modi
+ */
+@Entity
+@Table(name = "tbl_project_statistics_settings")
+@XmlRootElement
+@NamedQueries({
+        @NamedQuery(name = "ProjectStatisticsDTO.findAll", query = "SELECT p FROM ProjectStatisticsDTO p"),
+        @NamedQuery(name = "ProjectStatisticsDTO.findByProjectid", query = "SELECT p FROM ProjectStatisticsDTO p WHERE p.projectStatisticsDTOPK.projectid = :projectid"),
+        @NamedQuery(name = "ProjectStatisticsDTO.findByName", query = "SELECT p FROM ProjectStatisticsDTO p WHERE p.projectStatisticsDTOPK.projectid = :projectid and p.projectStatisticsDTOPK.name = :name"), })
+public class ProjectStatisticsDTO implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @EmbeddedId
+    protected ProjectStatisticsDTOPK projectStatisticsDTOPK;

```

Comment:
I am sorry, I did not get it. Does it mean I should remove primary key, foreign key concept over here?